Bottom: e16e386391ab3d9f1128f422c81560d225c291ee
Top:    75110ef0f4cf2076f1f4aa62b929796f1435d60e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-16 16:23:55 +0100

Use trackball to allow rotation of 3D view


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 23cca2d..0954882 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -124,6 +124,7 @@ a zoom in/out.
 #include "gui-icons-misc.data"
 #include "gui-trackball.h"
 #include "snavi.h"
+#include "gui-trackball.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
